README.md

```diff
@@ -424,6 +425,25 @@ async def handle_get():
     ]
 ```
 
+## Compression
+
```

Comment:
@naviot More passion ))))

Squall provides built-in blazing-fast compression based on Intel® Intelligent Storage Acceleration Library (Intel® ISA-L) using awesome Python's [isal](https://pypi.org/project/isal/) library as binding. 

Such in-app performance does game-changing opportunities for the entire system set up, read more here ....

Reply:
update doc